server/models/courseSchema.js

```diff
@@ -60,5 +92,10 @@ module.exports.pre('save', next => {
       this.sellingText_sv = sanitize(this.sellingText_sv)
     }
   }
+  // if (this.courseDisposition_en) {
```

Comment:
Remove? Or add for all fields?

Reply:
Fixed: 48c42b6f614c6973e79250e1355caebea26500e2